src/sif/autocost.cc

```diff
@@ -420,8 +423,9 @@ Cost AutoCost::EdgeCost(const baldr::DirectedEdge* edge,
           ? ferry_factor_
           : (edge->use() == Use::kRailFerry) ? rail_ferry_factor_ : density_factor_[edge->density()];
 
+  float speed_penalty = (edge_speed > top_speed_) ? (edge_speed - top_speed_) * 0.05f : 0.0f;
```

Comment:
> practically speaking i guess we could say we're driving a trabi and so we kind of want to avoid the autobahnen.

exactly. I did clone that from motorscooter, but think that's a very sensible thing to do. mostly I imagine you'd limit the top_speed for this exact kind of situation, tractor, trabi or tank ;) 0.05 _sounds_ reasonable, a penalty of 1 for 20 kmh difference. Should I leave a TODO for now?

Reply:
i think its fine to leave in for now, im just saying that `.05` may need to change. the main reason for that is because the factor in auto costing is in different units than the factor for motorscooter (probably) so the scales might not make sense to be the same. in any case we can tune it later on. no big deal